src/SonarQube.Client.Tests/SonarQubeService_HasOrganizations.cs

```diff
@@ -66,10 +66,10 @@ public void HasOrganizationsFeature_NotConnected()
             // No calls to Connect
             // No need to setup request, the operation should fail
 
-            Action action = () => { _ = service.HasOrganizations(CancellationToken.None).Result; };
+            Func<Task<bool>> func = async () => await service.HasOrganizations(CancellationToken.None);
 
-            action.Should().ThrowExactly<InvalidOperationException>()
-                .WithMessage("This operation expects the service to be connected.");
+            func.Should().ThrowExactly<InvalidOperationException>().And
+                .Message.Should().Be("This operation expects the service to be connected.");
```

Comment:
This was weird - the test failed when with the new package versions.

I experimented, and the old version of the code passed with `Fluent.Assertions` v5.6 and fails with v5.9.

All of the webclient tests have a `XXX_NotConnected` test. For some reason, this one was in a different form from all of the others. I've changed the test to match the others and it now passes.